src/runtime/lib/API.ts

```diff
@@ -76,7 +76,7 @@ export abstract class AbstractCommand<T> implements GenericCommand, Command {
   }
 
   private async _invoke(commandName: string, ...args: any[]): Promise<T> {
-    const typeArg = args.find((arg) => arg.CommandType);
+    const typeArg = args.find((arg) => arg && arg.CommandType);
```

Comment:
We could probably condense `arg && arg.CommandType` to `arg?.CommandType`

Great fix :)

Reply:
done